ground/src/main/java/com/google/android/ground/ui/map/MapController.kt

```diff
@@ -43,50 +50,51 @@ constructor(
   private val locationOfInterestRepository: LocationOfInterestRepository,
   private val surveyRepository: SurveyRepository,
   private val mapStateRepository: MapStateRepository,
-  @DefaultDispatcher private val defaultDispatcher: CoroutineDispatcher,
+  @ApplicationScope private val externalScope: CoroutineScope
 ) {
 
-  private val cameraUpdatesSubject: @Hot Subject<CameraPosition> = PublishSubject.create()
+  private val _cameraUpdates = MutableStateFlow<CameraPosition?>(null)
```

Comment:
Are we intending to treat this stream as a state container, or an event bus? If we're using `StateFlow` here, I'd assume the former, and that the field would be named something like `_cameraPosition`. If this is intended to model an event, then I assume we wouldn't want replay and a `MutableSharedFlow` would be more appropriate. Please cmiiw.